### PR TITLE
Add issue templates

### PR DESCRIPTION
**Pull request by joelostblom**
Here is a suggestion for what the issue templates could look like.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+about: 'Create a bug report to help us improve'
+labels: 'bug'
+---
+
+Type here...
+
+---
+
+Please follow these steps to make it more efficient to solve your issue:
+
```
Comment (jakevdp) [inline .github/ISSUE_TEMPLATE/bug-report.md:12]: We should add something here about including complete code to reproduce the problem

**Review by jakevdp — APPROVED** [review]
This is great, thanks!

File: .github/ISSUE_TEMPLATE/bug-report.md (hunk 1 above)
Comment (joelostblom) [reply .github/ISSUE_TEMPLATE/bug-report.md:12]: I do have it in the last check box below. Do you want me to put it on top instead?

Comment (jakevdp) [reply .github/ISSUE_TEMPLATE/bug-report.md:12]: Oh, I see. That should be fine!

**Pull request merged**